### PR TITLE
Contributor:Update approval-process-feature-overview.md 

### PR DESCRIPTION
**Pull request by VadymSachenko**
## PR Description
Original PR: https://github.com/spryker/spryker-docs/pull/1452

- Added a new note block to make it clear that the approver role is restricted by business unit.
- Edited an existing note block to clarify the meaning, as the original was confusing (it implied that _the Buyer role_ needs to set the permission, when in fact the permission needs to be set _for_ the buyer role.

## PR Description

TBD

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Review by andrewfobos777 — APPROVED** [review]

**Review by andriichekanov — APPROVED** [review]

File: docs/pbc/all/cart-and-checkout/approval-process-feature-overview.md [hunk 1]
```diff
@@ -18,6 +18,14 @@ The *Approval Process* feature lets B2B customers have multiple people contribut
 
 Permissions related to the approval process are configured based on the restrictions applied to a [company role](/docs/scos/user/features/{{site.version}}/company-account-feature-overview/company-user-roles-and-permissions-overview.html). Generally, the approval process is initiated when the cart total exceeds a certain amount set in the *Buy up to grand total* permissions. For example, an employee in a company may have to send their order to the manager for approval if the total order cost is above a certain amount. Only after the manager has received the request and approved the order, the employee can proceed to the checkout.
 
+{% info_block warningBox "Note" %}
+
+Approvers can only approve orders of employees *within their own business unit*.
+
+If there is an employee with a *Buy up to grand total* limit in a business unit that does not have any approvers, then this employee will not see any approvers they could send their order to at the checkout and thus will not be able to proceed with their order. Not even if another business unit of the same company does have an approver, and not even if that other business unit is the direct parent of the employee's business unit.
```
Comment (VadymSachenko) [inline docs/pbc/all/cart-and-checkout/approval-process-feature-overview.md:25]:
```suggestion
If there is an employee with a *Buy up to grand total* limit in a business unit that does not have any approvers, then this employee cannot see any approvers that they send their order to at the checkout and thus cannot proceed with their order. Not even if another business unit of the same company does have an approver and not even if that other business unit is the direct parent of the employee's business unit.
```

File: docs/pbc/all/cart-and-checkout/approval-process-feature-overview.md [hunk 2]
```diff
@@ -18,6 +18,14 @@ The *Approval Process* feature lets B2B customers have multiple people contribut
 
 Permissions related to the approval process are configured based on the restrictions applied to a [company role](/docs/scos/user/features/{{site.version}}/company-account-feature-overview/company-user-roles-and-permissions-overview.html). Generally, the approval process is initiated when the cart total exceeds a certain amount set in the *Buy up to grand total* permissions. For example, an employee in a company may have to send their order to the manager for approval if the total order cost is above a certain amount. Only after the manager has received the request and approved the order, the employee can proceed to the checkout.
 
+{% info_block warningBox "Note" %}
```
Comment (andriitserkovnyi) [inline docs/pbc/all/cart-and-checkout/approval-process-feature-overview.md:21]: Come up with a name for this note

Comment (VadymSachenko) [reply docs/pbc/all/cart-and-checkout/approval-process-feature-overview.md:21]: @andriitserkovnyi Not sure about the title. We can cut out and paste the first sentence:

```suggestion
{% info_block warningBox "Approvers can only approve orders within their own business unit" %}
```

Comment (andriitserkovnyi) [reply docs/pbc/all/cart-and-checkout/approval-process-feature-overview.md:21]:
```suggestion
{% info_block warningBox "Approvals within a business unit" %}
```

File: docs/pbc/all/cart-and-checkout/approval-process-feature-overview.md [hunk 3]
```diff
@@ -18,6 +18,14 @@ The *Approval Process* feature lets B2B customers have multiple people contribut
...
+
+Approvers can only approve orders of employees *within their own business unit*.
+
+If an employee with a *Buy up to grand total* limit in a business unit is without any approvers, at the checkout, the employee can't see any approvers to send their order to and thus can't proceed with their order. Not even if another business unit of the same company does have an approver. And not even if that other business unit is the direct parent of the employee's business unit.
```
[the remark this answers is not in the document]
Comment (andriitserkovnyi) [reply docs/pbc/all/cart-and-checkout/approval-process-feature-overview.md:25]:
```suggestion
If an employee with a *Buy up to grand total* limit is in a business unit without any approvers, at the checkout, the employee can't see any approvers to send their order to and thus can't proceed with their order. Not even if another business unit of the same company does have an approver. And not even if that other business unit is the direct parent of the employee's business unit.
```

**Review by andriitserkovnyi — APPROVED** [review]

**Pull request merged**